packages/rocketchat-i18n/i18n/en.i18n.json

```
@@ -4477,6 +4477,9 @@
   "User_joined_channel_female": "Has joined the channel.",
   "User_joined_channel_male": "Has joined the channel.",
   "User_joined_conversation": "Has joined the conversation",
+  "user_joined_otr": "Has joined OTR chat.",
```

Please use the same pattern.
```suggestion
  "User_joined_otr": "Has joined OTR chat.",
  "User_key_refreshed_successfully": "key refreshed successfully",
  "User_requested_otr_key_refresh": "Has requested key refresh.",
```

---

Done!